mobile/src/main/java/ch/epfl/sdp/mobile/ui/game/GameScreen.kt

```diff
@@ -1,79 +1,148 @@
-import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.lazy.LazyRow
-import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
 import androidx.compose.foundation.shape.RoundedCornerShape
-import androidx.compose.material.MaterialTheme
-import androidx.compose.material.Text
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.*
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.Alignment
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.key
+import androidx.compose.ui.Alignment.Companion.CenterVertically
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.unit.dp
+import ch.epfl.sdp.mobile.state.LocalLocalizedStrings
+import ch.epfl.sdp.mobile.ui.BlackKing
+import ch.epfl.sdp.mobile.ui.ChessIcons
+import ch.epfl.sdp.mobile.ui.PawniesColors
+import ch.epfl.sdp.mobile.ui.PawniesColors.Green500
+import ch.epfl.sdp.mobile.ui.PawniesColors.Green800
+import ch.epfl.sdp.mobile.ui.PawniesColors.Orange200
+import ch.epfl.sdp.mobile.ui.WhiteKing
+import ch.epfl.sdp.mobile.ui.game.ChessBoardState.Color.White
+import ch.epfl.sdp.mobile.ui.game.GameScreenState.Message
+import ch.epfl.sdp.mobile.ui.game.GameScreenState.Move
+import com.google.accompanist.flowlayout.FlowRow
 
 /**
  * This screen display an ongoing game of chest
  *
  * @param state the [GameScreenState] that manage the composable contents
  * @param modifier the [Modifier] for the composable
+ * @param contentPadding the [PaddingValues] for this composable.
  */
 @Composable
 fun <Piece : ChessBoardState.Piece> GameScreen(
     state: GameScreenState<Piece>,
     modifier: Modifier = Modifier,
+    contentPadding: PaddingValues = PaddingValues(),
 ) {
-  Column(modifier) {
-    TitleText(text = "Chess game")
-    MoveList(state.moves)
-    ChessBoard(state, modifier = Modifier.padding(16.dp))
-    TitleText(text = "VR / Voice Recognition")
-  }
+  Scaffold(
+      modifier = modifier,
+      topBar = {
+        GameScreenTopBar(
+            onBackClick = state::onBackClick,
+            onArClick = state::onArClick,
+            onListenClick = state::onListenClick,
+            listening = state.listening,
+            modifier = Modifier.fillMaxWidth(),
+        )
+      },
+      content = { scaffoldPadding ->
+        Column(
+            modifier =
+                modifier
+                    .verticalScroll(rememberScrollState())
+                    .padding(contentPadding)
+                    .padding(scaffoldPadding)
+                    .padding(start = 16.dp, end = 16.dp, top = 48.dp, bottom = 48.dp),
+            verticalArrangement = Arrangement.spacedBy(32.dp),
+        ) {
+          Column {
+            Player(White, state.white.name, state.white.message)
+            Player(ChessBoardState.Color.Black, state.black.name, state.black.message)
```

Comment:
Not consistent with the line above

Reply:
Fixed in ad8c052ba7d095a4a546dfddaad7be62362913c8.